OrbitVulkanLayer/PhysicalDeviceManagerTest.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "DeviceManager.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using ::testing::Return;
+
+namespace orbit_vulkan_layer {
+
+class MockDispatchTable {
+ public:
+  MOCK_METHOD(PFN_vkGetPhysicalDeviceProperties, GetPhysicalDeviceProperties,
+              (VkPhysicalDevice dispatchable_object));
+};
+
+TEST(PhysicalDeviceManager, AnUnrackedDeviceCannotBeQuerried) {
```

Comment:
I think that this file and all the `PhysicalDeviceManager` in `TEST` should now be renamed to `DeviceManager` for consistency with the name of the class.

Reply:
Yep, thanks!